src/docs/asciidoc/getting-started-eventuate-tram-sagas.adoc

```diff
@@ -62,20 +62,48 @@ repositories {
 
 If you are writing a Saga orchestrator add this dependency to your project:
 
-* `io.eventuate.tram.sagas:eventuate-tram-sagas-simple-dsl:$eventuateTramSagasVersion`
+===== Spring
+
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-spring-orchestration-simple-dsl:$eventuateTramSagasVersion`
+
+===== Micronaut
+
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-orchestration-simple-dsl:$eventuateTramSagasVersion`
```

Comment:
Why `eventuate-tram-sagas-orchestration-simple-dsl`? Isn't it a transitive dependency of `eventuate-tram-sagas-micronaut-orchestration`  (like with Spring?)

Reply:
Chris, no, eventuate-tram-sagas-micronaut-orchestration and eventuate-tram-sagas-spring-orchestration do not have eventuate-tram-sagas-orchestration-simple-dsl as dependency. eventuate-tram-sagas-spring-orchestration-simple-dsl does. But we have no eventuate-tram-sagas-micronaut-orchestration-simple-dsl module. Should I create it.

Reply:
Got it. Yes. Create eventuate-tram-sagas-micronaut-orchestration-simple-dsl so that it mirrors  the Spring version?

Reply:
Ok, will do